ADR/Components/Aerodynamic_components/tests/test_HS.py

```diff
@@ -0,0 +1,49 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import (
+    Aerodynamic_surface,
+)
+from ADR.Components.Aerodynamic_components.HS import HS
+import numpy.testing as npt
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def aerodynamic_surface():
+    aerodynamic_surface_parameters = {
+        "airfoil_clmax": 2.5,
+        "airfoil1_name": "s1223",
+        "airfoil2_name": "s1223",
+        "airfoil3_name": "s1223",
+        "span1": 2,
+        "span2": 1.5,
+        "chord1": 0.40,
+        "chord2": 0.30,
+        "chord3": 0.35,
+        "twist1": 0,
+        "twist2": 0,
+        "twist3": 0,
+        "incidence": 0,
+        "x": 1,
+        "z": 1,
+    }
+    aerodynamic_surface = HS(aerodynamic_surface_parameters)
+    return aerodynamic_surface
+
+
+def test_calc_area(aerodynamic_surface):
+    aerodynamic_surface.calc_area()
+    new_area = aerodynamic_surface.area
+    npt.assert_almost_equal(new_area, 2.375, decimal=3)
+
+
+def test_update_alpha(aerodynamic_surface):
+    aerodynamic_surface.update_alpha(15)
+    new_alpha = aerodynamic_surface.attack_angle
+    npt.assert_almost_equal(new_alpha, -15)
+
+
+def test_set_incidence_range(aerodynamic_surface):
+    aerodynamic_surface.set_incidence_range(-5, 15)
+    incidence_range = aerodynamic_surface.incidence_range
+    x = np.arange(-15, 26, 1)
+    npt.assert_almost_equal(incidence_range, x)
```

Comment:
The tests are perfect. They will ensure for now that you don't break the code that is currently working.

For the long term we should definitely refactor those functions and make them functional. The "set_incidence_range()", for example, has it's result dependent from the calculated stall angles of the surface. If we instantiate the class with an airfoil other than selig1223, the test would fail.

Can you open an Issue listing the functions on HS that should be refactored in the future?